### ISSUE TITLE
About AVA dataset and lebels

### ISSUE THREAD

**Issue by huanmofazong** [fei-hdu/ReLIC#2]
May I ask some questions about the dataset？

IN https://github.com/mtobeiyf/ava_downloader  have many links, I uses  **torrent file**  download the dataset, but there seems to be something wrong with the picture.May I ask which link you downloaded from?

This  AVA dataset should only have training sets and test sets, but in your code there are training sets, test sets, and validation sets.And I find that there seem to be different versions of the division in guthub , may I ask where this label file comes from?
                                                                                                                       
Thank you for your reply and wish you academic success
